src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+export const Person = ({
+  person:
+  { name,
+    age,
+    sex,
+    isMarried,
+    partnerName },
+}) => (
```

Comment:
```suggestion
export const Person = ({
  person: { 
    name,
    age,
    sex,
    isMarried,
    partnerName, 
}}) => (
```
Pay attention to the formatting